alfresco-integration/src/main/java/com/github/dynamicextensionsalfresco/osgi/spring/WebApplicationSystemPackageFactoryBean.java

```diff
@@ -47,8 +53,16 @@ public boolean isSingleton() {
 	}
 
 	@Override
-	public Set<SystemPackage> getObject() throws Exception {
-		return createSystemPackages();
+	public Set<SystemPackage> getObject() {
+		final long start = System.currentTimeMillis();
+		Set<SystemPackage> systemPackages = createSystemPackages();
+		final long finish = System.currentTimeMillis();
+		if (logger.isInfoEnabled() && (finish - start) > 1500L) {
```

Comment:
There is no need to wrap this check into `logger.isInfoEnabled()`.
In general, checking if a log level is enabled is only used when doing expensive operations to create log messages.